packages/api/src/schema/ai-api-schema.yaml

```diff
@@ -0,0 +1,869 @@
+          default: SG161222/RealVisXL_V4.0_Lightning
+        prompt:
+          type: string
+          title: Prompt
+          description: >-
+            Text prompt(s) to guide image generation. Separate multiple prompts
+            with '|' if supported by the model.
+        height:
+          type: integer
+          title: Height
+          description: The height in pixels of the generated image.
+          default: 576
+        width:
+          type: integer
+          title: Width
+          description: The width in pixels of the generated image.
+          default: 1024
+        guidance_scale:
+          type: number
+          title: Guidance Scale
+          description: >-
+            Encourages model to generate images closely linked to the text
+            prompt (higher values may reduce image quality).
+          default: 7.5
+        negative_prompt:
+          type: string
+          title: Negative Prompt
+          description: >-
+            Text prompt(s) to guide what to exclude from image generation.
+            Ignored if guidance_scale < 1.
+          default: ''
+        safety_check:
+          type: boolean
+          title: Safety Check
+          description: >-
+            Perform a safety check to estimate if generated images could be
+            offensive or harmful.
+          default: true
+        seed:
+          type: integer
+          title: Seed
+          description: Seed for random number generation.
+        num_inference_steps:
+          type: integer
+          title: Num Inference Steps
+          description: >-
+            Number of denoising steps. More steps usually lead to higher quality
+            images but slower inference. Modulated by strength.
+          default: 50
+        num_images_per_prompt:
+          type: integer
+          title: Num Images Per Prompt
+          description: Number of images to generate per prompt.
+          default: 1
+      type: object
+      required:
+        - prompt
+      title: TextToImageParams
+      additionalProperties: false
+    ValidationError:
+      properties:
+        loc:
+          items:
+            anyOf:
+              - type: string
+              - type: integer
+          type: array
+          title: Location
+        msg:
+          type: string
+          title: Message
+        type:
+          type: string
+          title: Error Type
+      type: object
+      required:
+        - loc
+        - msg
+        - type
+      title: ValidationError
+    VideoResponse:
+      properties:
+        images:
+          items:
+            $ref: '#/components/schemas/Media'
+          type: array
+          title: Images
+          description: The generated images.
+      type: object
+      required:
+        - images
+      title: VideoResponse
+      description: Response model for image generation.
+    chunk:
+      properties:
+        timestamp:
+          items: {}
```

Comment:
Not sure I ever encountered this type, is `{}` just defining a generic object? could it be defined specifically as an array of numbers with [number]?

Reply:
This is defined in `ai-worker` repository, we only clone it here. Any more detailed specification should go there.